### PR TITLE
User generated

### PR DESCRIPTION
**Pull request by jspellman**
All the changes from AGWG meeting of 10 August 2021 for Conformance and Glossary.  I am including the Outcome (mostly because I don't know how to remove it and I don't want to lose the work done). I did not make changes to the Outcome.

### CODE REVIEW

**Review by michael-n-cooper — DISMISSED** [review]

**Review by bruce-usab — APPROVED** [review]
I am all for this edit going live.

I saw the not that <q>Future work on the glossary will better define terms such as publisher, content author, etc.</q>

Also for the glossary, the convention with 2.x is that definitions can function as copy/paste for the terms.  This version does not work for that.

File: guidelines/index.html [hunk 1]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+			<p>An important part of WCAG Conformance is the specific guidance that is associated with individual WCAG 3 guidelines and outcomes. Not all WCAG 3 guidelines will have unique outcomes and testing for User Generated Content. Unless User Generated Content requirements are specified in a particular guideline, that guideline applies as written whether or not the content is User Generated. </p>
+			
+			<p class="ednote">We plan for the Q4 Working Draft to include specific examples of guidelines with additional requirements for user generated content.  One example would be alternative text. The Authoring Tool Accessibility Guidelines (ATAG) has specific guidance for providing a mechanism for alternative text. The <a href="https://www.w3.org/TR/ATAG20/#gl_b23">ATAG  2.0 Guideline B.2.3</a> - "Assist authors with managing alternative content for non-text content" could be adapted to provide specific, guideline-related guidance for user generated alternative text. </p>
```
Comment (patrickhlauke) [inline guidelines/index.html:921]: `The Authoring Tool Accessibility Guidelines (ATAG) has specific guidance` > `The Authoring Tool Accessibility Guidelines (ATAG) have specific guidance` (have, plural)

Comment (patrickhlauke) [reply guidelines/index.html:921]: stray unnecessary space before the closing `</p>`

File: guidelines/index.html [hunk 2]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+					<li>uploaded videos or other multimedia.</li>
+				</ul>
+			<p>User Generated Content is provided for publication by visitors where the content platform specifically welcomes and encourages it. User-generated content is content that is submitted through a user interface designed specifically for members of the public and customers. Use of the same user interface as an authoring tool for publication of content by agents of the publisher (such as employees, contractors, or authorized volunteers) acting on behalf of the publisher does not make that content User Generated Content. The purpose of the User Generated Content Conformance is to allow WCAG 3 outcomes and methods to require additional or different steps to improve the accessibility of User Generated Content.</p>
+			<p>An important part of WCAG Conformance is the specific guidance that is associated with individual WCAG 3 guidelines and outcomes. Not all WCAG 3 guidelines will have unique outcomes and testing for User Generated Content. Unless User Generated Content requirements are specified in a particular guideline, that guideline applies as written whether or not the content is User Generated. </p>
```
Comment (patrickhlauke) [inline guidelines/index.html:919]: stray unnecessary space before the closing `</p>`

File: guidelines/index.html [hunk 3]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+			
+			<p class="ednote">We plan for the Q4 Working Draft to include specific examples of guidelines with additional requirements for user generated content.  One example would be alternative text. The Authoring Tool Accessibility Guidelines (ATAG) has specific guidance for providing a mechanism for alternative text. The <a href="https://www.w3.org/TR/ATAG20/#gl_b23">ATAG  2.0 Guideline B.2.3</a> - "Assist authors with managing alternative content for non-text content" could be adapted to provide specific, guideline-related guidance for user generated alternative text. </p>
+			
+			<p>The web content publisher should identify all locations of User Generated Content (such as commentary on hosted content, product descriptions for consumer to consumer for sale listings, and restaurant reviews) and perform standard accessibility evaluation analysis for each. If there are no accessibility issues, the User Generated Content is fully conforming.</p>
```
Comment (patrickhlauke) [inline guidelines/index.html:923]: `product descriptions for consumer to consumer for sale listings` sounds a bit dense. maybe `product descriptions for consumer-to-consumer sale listings`

File: guidelines/index.html [hunk 4]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+			<section>
+			<h4>Steps to Conform</h4>
+			<p>If accessibility issues are identified, or if the website author wants to proactively address potential accessibility issues that might arise from User Generated Content, then all of the following must be indicated alongside the User Generated Content or in an Accessibility Statement published on the site or product that is linked from the view or page in a consistent location:</p>
+			<ol>
+				<li>Clearly identify where User Generated Content can be found on the publisher's digital product (perhaps by id href);</li>
```
Comment (patrickhlauke) [inline guidelines/index.html:928]: `perhaps by id href` sounds unnecessarily technical. who would this be aimed at? regular public user facing an accessibility issue won't know what to do with an idref, or XPath, or similar. this sounds like something only an automated tool vendor might have some interest in...

File: guidelines/index.html [hunk 5]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+				<li>Clearly identify the steps taken to encourage accessibility in User Generated Content such as prompting the user for ALT text for their uploaded images before they are accepted and the disallowal of text attributes except as they are part of semantic markup such as strong, headings, etc., as enumerated in Guideline Outcomes;</li>
```
Comment (patrickhlauke) [inline guidelines/index.html:929]: `before they are accepted and the disallowal of text attributes except as they are part of semantic markup such as strong, headings, etc., as enumerated in Guideline Outcomes`

`and the disallowal` sounds awkward and unusual. perhaps `...accepted, or only allowing the use of semantic structural markup (and filtering out presentational markup for styling) in text entry fields, etc.`

**Review by detlevhfischer — APPROVED** [review]
The changes suggested by Patrick look all good to me.

File: guidelines/index.html (hunk 1 above)
Comment (jspellman) [reply guidelines/index.html:921]: Sorry, WAI editorial rules are that all the Guidelines are singular.

File: guidelines/index.html [hunk 6]
```diff
@@ -906,6 +906,31 @@ <h4>Gold</h4>
...
+			
+			<p>The web content publisher should identify all locations of User Generated Content (such as commentary on hosted content, product descriptions for consumer to consumer for sale listings, and restaurant reviews) and perform standard accessibility evaluation analysis for each. If there are no accessibility issues, the User Generated Content is fully conforming.</p>
+			<section>
+			<h4>Steps to Conform</h4>
+			<p>If accessibility issues are identified, or if the website author wants to proactively address potential accessibility issues that might arise from User Generated Content, then all of the following must be indicated alongside the User Generated Content or in an Accessibility Statement published on the site or product that is linked from the view or page in a consistent location:</p>
```
[the remark this answers is not in the document]
Comment (jspellman) [reply guidelines/index.html:926]: Even though I agree with Pat, that is a substantial change at this point and we should note it for a future draft.  Unless you all want to go back to the group to debate it.

**Pull request merged**